src/plugins/remote-loader/index.ts

```diff
@@ -14,7 +14,7 @@ export interface RemotePlugin {
   /** The UMD/global name the plugin uses. Plugins are expected to exist here with the `PluginFactory` method signature */
   libraryName: string
   /** The settings related to this plugin. */
-  settings: JSONValue
+  settings: JSONObject
```

Comment:
I checked how we build the settings that get served by the CDN and it's always an object.